packages/server/lib/browsers/chrome.ts

```diff
@@ -318,8 +317,124 @@ const _handleDownloads = async function (client, dir, automation) {
   })
 }
 
-const _setAutomation = async (client: CRIWrapper.Client, automation: Automation, closeCurrentTarget: () => Promise<void>) => {
-  const cdpAutomation = await CdpAutomation.create(client.send, client.on, closeCurrentTarget, automation)
+let frameTree
+let gettingFrameTree
+
+const onReconnect = (client: CRIWrapper.Client) => {
+  // if the client disconnects (e.g. due to a computer sleeping), update
+  // the frame tree on reconnect in cases there were changes while
+  // the client was disconnected
+  return _updateFrameTree(client, 'onReconnect')()
+}
+
+// eslint-disable-next-line @cypress/dev/arrow-body-multiline-braces
+const _updateFrameTree = (client: CRIWrapper.Client, eventName) => async () => {
```

Comment:
see [original]()https://github.com/cypress-io/cypress/pull/18075/files#diff-e2888f195a1c64929f7e4de7cb14eba872a682eb1cd29f8d50b87131c1f78c1aR344 in `9.x`